src/components/QrScanner/ValidityCheckScreenContainer.test.tsx

```diff
@@ -42,7 +48,10 @@ describe("ValidityCheckScreenContainer", () => {
 
   it("should navigate when document is valid", () => {
     expect.assertions(1);
-    mockUseVerifier.mockReturnValue({ overallValidity: CheckStatus.VALID });
+    mockUseVerifier.mockReturnValue({
+      statuses: { overallValidity: CheckStatus.VALID },
+      verify: jest.fn()
+    });
 
     render(<ValidityCheckScreenContainer navigation={mockNavigation} />);
 
```

Comment:
should we now include a test that checks that `verify` is called when the component mounts?